specification/gedcom.md

```diff
@@ -402,16 +402,63 @@ as described in [Extensions].
 
 A **standard structure** is a structure whose type, tag, meaning, superstructure, and cardinality within the superstructure are described in this document. This includes records such as `INDI` and substructures such as `INDI`.`NAME`.
 
-Two forms of **extension structures** are permitted:
+A **tagged extension structure** is a structure whose tag matches production `extTag`. Tagged extension structures may appear as records or substructures of any other structure. Their meaning is defined by their tag, as is discussed more fully in the section [Extension Tags].
 
-- A **tagged extension structure** is a structure whose tag matches production `extTag`. Tagged extension structures may appear as records or substructures of any other structure.
-- An **extended-use standard structure** is a structure whose type, tag, and meaning are defined in this document and whose superstructure is a tagged extension structure.
+Any substructure of a tagged extension structure that uses a tag matching `stdTag` is an **extension-defined substructure**.
+Substructures of an extension-defined substructure that uses a tag matching `stdTag` are also extension-defined substructures.
+The meaning and use of each extension-defined substructure is defined by the tagged extension structure it occurs within, not by its tag alone nor by this specification.
 
-Extension structures may have substructures, which may be either tagged extension structures of extended-use standard structures.
+:::example
+In the following
+
+```gedcom
+0 @P1@ _LOC
+1 NAME Βυζάντιον
+2 DATE FROM 667 BCE TO 324
+1 _POP 15149358
+2 DATE 31 DEC 2020
+0 @I1@ INDI
+1 BIRT
+2 _LOC @P1@
+```
+
+- Both `_LOC` are tagged extension structures, as is `_POP`.
+- `_LOC`.`NAME` and `_LOC`.`NAME`.`DATE` are both extension-defined substructures. Their meaning is defined by the specification defining `_LOC`.
+- `_POP`.`DATE` is an extension-defined substructure. Its meaning is defined by the specification defining `_POP`.
+- Even though both `DATE`s appear to have `g7:type-DATE` payloads, we can't know that is the intended datatype without consulting the defining specifications of `_LOC` and `_POP`, respectively.
```

Comment:
My view as an implementer is this is not good for implementation.   I'd strongly prefer to keep standard names as having the datatypes in the spec.

Reply:
Good point. Example text updated.